_topic_map.yml

```diff
@@ -165,7 +165,7 @@ Topics:
   File: generate-diagnostic-bundle
 - Name: Configuring endpoints
   File: configure-endpoints
-- Name: Monitoring with Prometheus
```

Comment:
Why did you change this?

Reply:
I though the prometheus is implementation detail of OpenShift Monitoring